Ginger/GingerCore/GeneralLib/General.cs

```diff
@@ -906,6 +906,48 @@ public static T TryFindParent<T>(DependencyObject child)
             else
                 return TryFindParent<T>(parentObject);
         }
+
+        /// <summary>
+        /// This function is used to replace the password value from the string with *****
+        /// </summary>
```

Comment:
need to mention it also will remove all spaces... that probably the issues with this method to be generic for now

Reply:
We are not removing password from the original connection string, but we will face issue when we have space in the password itself